src/vxworks/mod.rs

```diff
@@ -1976,16 +2027,14 @@ extern "C" {
     pub fn sigqueue(
         __pid: pid_t,
         __signo: ::c_int,
-        __value: ::size_t, // Actual type is const union sigval value,
-                           // which is a union of int and void *
+        __value: ::sigval,
     ) -> ::c_int;
 
     // signal.h for user
     pub fn _sigqueue(
         rtpId: ::RTP_ID,
         signo: ::c_int,
-        pValue: *mut ::size_t, // Actual type is const union * sigval value,
-        // which is a union of int and void *
+        pValue: *mut ::sigval,
```

Comment:
Actually in VxWorks this is 

` const union sigval * pValue`

so this should be

`pValue: *const ::sigval`

(the code in rtp.rs incorrectly uses mut too ...)